.github/workflows/review.yaml

```diff
@@ -0,0 +1,101 @@
+#
+# Check for the Squid Project Merg Process criteria
+# published at https://wiki.squid-cache.org/MergeProcedure#votes
+#
+name: PR Approval
+
+on:
+  schedule:
+    - cron: "10 1 * * *" # once a day
+
+  pull_request:
+    # test PRs targeting this branch code
+    branches: [ "master" ]
+
+  # allows to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+concurrency:
+  # Cancel ongoing tests in case of push to the same PR or staging branch,
+  # but let previous master commit tests complete.
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: ${{ github.ref != 'refs/heads/master' }}
+
+env:
+  # empty except for pull_request events
+  PULL_REQUEST_NUMBER: ${{ github.event.pull_request.number }}
+
+  # required for use of gh CLI tool
+  GH_TOKEN: ${{ github.token }}
+
+jobs:
+
+  Review:
+    runs-on: ubuntu-24.04
+
+    steps:
+      - name: Checkout Squid sources
```

Comment:
We are missing one step: this file will be in all forks of the squid repository, and apply to them as well, which we do not want.
Please protect against this ([instructions](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/using-conditions-to-control-job-execution#example-only-run-job-for-specific-repository))